src/components/Nav/components/NavLinks/index.jsx

```diff
@@ -26,31 +26,30 @@ const GordonNavLinks = ({ onLinkClick }) => {
   };
 
   const dialogBox = () => {
-    if (dialog === 'offline') {
-      return (
-        <GordonDialogBox
-          open={dialog}
-          onClose={() => setDialog(null)}
-          title={'Offline Mode'}
-          buttonClicked={() => setDialog(null)}
-          buttonName={'Okay'}
-        >
-          This feature is unavailable offline. Please reconnect to internet to access this feature.
-        </GordonDialogBox>
-      );
-    } else if (dialog === 'unauthorized') {
-      return (
-        <GordonDialogBox
-          open={dialog}
-          onClose={() => setDialog(null)}
-          title={'Credentials Needed'}
-          buttonClicked={() => setDialog(null)}
-          buttonName={'Okay'}
-        >
-          This feature is unavailable while not logged in. Please log in to access it.
-        </GordonDialogBox>
-      );
-    }
+    let message, title;
+    switch (dialog) {
+      case 'offline':
+        message = 'That page is not available offline. Please reconnect to internet to access it.';
+        title = 'Unavailable Offline';
+      case 'unauthorized':
+        message = 'That page is only available to authenticated users. Please log in to access it.';
+        title = 'Unavailable Offline';
```

Comment:
```suggestion
      case 'offline':
        message = 'That page is not available offline. Please reconnect to internet to access it.';
        title = 'Unavailable Offline';
        break;
      case 'unauthorized':
        message = 'That page is only available to authenticated users. Please log in to access it.';
        title = 'Unavailable Offline';
        break;
```
Why no breaks here but one at the bottom? Wouldn't all of these end up cascading down to the default case if there is no break?